src/platform/components/Header/index.module.scss

```diff
@@ -4,6 +4,7 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+  width: 100%;
```

Comment:
No need to set the width to 100%

Reply:
tried to remove it but it corrupted the design.. nothing working but this way.